### PR TITLE
Add local environment notes to README.md

### PR DESCRIPTION
**Pull request by jraddaoui**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,5 +21,147 @@ fault-tolerant workflows suited for preservation.
 
 Visit https://enduroproject.netlify.com for more details.
 
-[National Health Archive (NHA)]: https://www.piql.com/norwegians-digital-health-data-to-be-preserved-for-future-generations/
-[National Center for Truth and Reconciliation (NCTR)]: https://nctr.ca/about/about-the-nctr/our-mandate/
+## Local environment
+
+Enduro uses Tilt to set up a local environment building the Docker images in a
```
Comment (replaceafill) [inline README.md:26]: I think a link to https://tilt.dev/ would be nice here.

File: README.md [hunk 2]
```diff
@@ -21,5 +21,147 @@ fault-tolerant workflows suited for preservation.
...
+While the Docker images are built/downloaded and the Kubernetes resources are
+created, hit `space` to open the Tilt UI in your browser. Check the [Tilt UI]
+documentation to know more about it.
+
+### Access
+
+There are four services available from the host:
+
+- Enduro dashboard: http://localhost:3000
+- Minio console: http://localhost:7460 (minio/minio123)
```
Comment (replaceafill) [inline README.md]: I think it'd be worth to explain that `minio/minio123` are the default credentials to the service.

File: README.md [hunk 3]
```diff
@@ -21,5 +21,147 @@ fault-tolerant workflows suited for preservation.
...
+- Temporal UI: http://localhost:7440
+- Opensearch dashboards: http://localhost:7500
+
+### Live updates
+
+Tilt will watch for file changes in the project folder and it will sync those
+changes, rebuild the Docker images and recreate the resources when necessary.
+The `enduro-dashboard` uses Vite to serve the application in development with
+hot reload, while the Go services require to recreate the images to build the
+binaries and they take longer to update.
```
Comment (replaceafill) [inline README.md]: I think you want to explain that the dashboard hot reload is fast and the Go rebuilds are not, but I find this a bit difficult to parse/understand.

File: README.md [hunk 4]
```diff
@@ -21,5 +21,147 @@ fault-tolerant workflows suited for preservation.
...
+
+Additionally, there are refresh buttons on each resource in the Tilt UI that
+allows to trigger manual updates and re-execute jobs and local resources.
```
Comment (replaceafill) [inline README.md]: ESL here: should this be `allow`?

**Review by replaceafill — APPROVED** [review]
LGTM. Only minor comments for you to consider.

**Review by sbreker — APPROVED** [review]
Looks great @jraddaoui :+1:

**Pull request merged**